### PR TITLE
iOS: Integrate logic for apple wallet button

### PR DESCRIPTION
**Pull request by lukewalczak**

### CODE REVIEW

File: app/MMB/src/scenes/appleWallet/WalletContent.js [hunk 1]
```diff
@@ -2,11 +2,29 @@
...
 import DateAndTime from './DateAndTime';
+import WalletContext from '../../context/WalletContext';
+
+const createAddHandler = pkpassUrl => async () => {
+  try {
+    await PassBook.addPass(pkpassUrl);
+  } catch (err) {
+    return Alert.translatedAlert(undefined, {
+      passThrough:
+        'There was an error during communication with our server. Please try again later',
```
Comment (tbergq) [inline app/MMB/src/scenes/appleWallet/WalletContent.js]: You need to create a translation for this in `ManageMyBooking.js`

Comment (lukewalczak) [reply app/MMB/src/scenes/appleWallet/WalletContent.js]: 🤦‍♂️

**Comment by tbergq** [comment]
Also, I think we have to: 

A: Add a scroll view, or 
B: Expand the height of the view

image

**Comment by lukewalczak** [comment]
I am more inclined to use `ScrollView`

**Comment by tbergq** [comment]
I did a quick check on an iPhone 5 (the smaller one) it seems what they do is expand the view, and reduce the image.

**Comment by tbergq** [comment]
We should be able to do this by replacing the hardcoded heights with flex I guess

**Comment by lukewalczak** [comment]
Can you please attach the screenshot from iPhone 5?

**Comment by tbergq** [comment]
it is so pretty 🙈 
file

**Pull request merged**